ui/hooks/useCurrencyRatePolling.ts

```diff
@@ -16,9 +16,10 @@ const useCurrencyRatePolling = (networkClientId?: string) => {
   const selectedNetworkClientId = useSelector(getSelectedNetworkClientId);
 
   usePolling({
-    startPollingByNetworkClientId: currencyRateStartPollingByNetworkClientId,
+    startPolling: (input) =>
+      currencyRateStartPollingByNetworkClientId(input.networkClientId),
```

Comment:
Can you help me understand why we aren't updating the action to also be `currencyRateStartPolling`?

Reply:
in actions.ts i kept the function name `currencyRateStartPollingByNetworkClientId` to minimize changes since it does still accept a network client id (for now)